### ISSUE TITLE
Unable to run forwarding step on test set

### ISSUE THREAD

**Issue by kevinmchu** [mravanelli/pytorch-kaldi#253]
I'm running the TIMIT LSTM on custom features, and was able to successfully train a model. Now I'm testing the model using a custom test set, but I'm experiencing an issue where the test features do not get forwarded through the trained model. Here is the terminal output:

```
- Reading config file......OK!
- Chunk creation......OK!

Testing TIMIT_test chunk = 1 / 1
Decoding TIMIT_test output out_dnn2
kaldi_decoding_scripts//decode_dnn.sh xx
```

Additionally, the log file only indicates that hmm-info and ali-to-pdf were run, but there are no errors or warnings listed. I suspect this issue has to do with modifying the phonemap file, score.sh, and timit_norm_trans.pl to map to 41 phones rather than 40 phones. I reverted back to using 40 phones, but the features never appear to be forwarded and the code seems to be stuck at the decoding phase.

Do you have any suggestions on how to solve this issue?
